xml/obs_build_config.xml

```diff
@@ -1144,5 +1144,92 @@ Support: pax debbuild</screen>
    </para>
    <screen>%_hardened_build 0</screen>
   </sect2>
+
+  <sect2 xml:id="sec-prjconfig-macros-predefined">
+   <title>Predefined Macros by OBS</title>
+   <para>
+    OBS is predefining the following macros when concatenating the build config
+    snippets from the used projects. These can be used for conditions inside
+    of the build config or in any build description at build time.
+   </para>
+
+  <itemizedlist>
+   <listitem>
+    <formalpara>
+     <title>%_project</title>
+     <para>
+       This macro is only set once at the beginning to the project name 
+       used for building.
+       This macro is available in the build configuration context and
+       during build time.
+     </para>
+    </formalpara>
+   </listitem>
+   <listitem>
+    <formalpara>
+     <title>%_repository</title>
+     <para>
+       This macro is set to the repository name of every build configuration snippet.
+       This macro is available in the build configuration context and
+       during build time.
+     </para>
+    </formalpara>
+   </listitem>
+   <listitem>
+    <formalpara>
+     <title>%_is_this_project</title>
+     <para>
+       This macro is only set to 1 for the build configuration
+       snippet from the acutal build project. It is set to 0
+       otherwise. This macro did not exist before OBS 3.0.
+       This macro is only available in the build configuration context.
+     </para>
+    </formalpara>
+   </listitem>
+   <listitem>
+    <formalpara>
+     <title>%_is_in_project</title>
+     <para>
+       This macro is set to 1 for the build configuration
+       snippet from the actual build project. In addition it
+       is also set to 1 when the project name is a namespace of
+       the building project. Eg. when building in home:user:subproject 
+       against a path of home:user it is set to 1 for both of them. 
+       It is set to 0 otherwise. This macro did not exist before OBS 3.0.
+       This macro is only available in the build configuration context.
+     </para>
+    </formalpara>
+   </listitem>
+   <listitem>
+    <formalpara>
+     <title>%_download_url</title>
+     <para>
+       This macro is set to string defined as base download url in the
+       the OBS instance by the admin.
+       This macro is only available during build time.
+     </para>
+    </formalpara>
+   </listitem>
+   <listitem>
+    <formalpara>
+     <title>%distribution</title>
+     <para>
+       This macro is set to "PROJECT / REPOSITORY" string. The REPOSITORY
+       is skipped in case it is set to "standard".
+       This macro is only available during build time.
```

Comment:
```suggestion
       This macro is set to <code>%_project / %_repository</code>.
       If the build happens in the project with the name <emphasis>home:hans</emphasis> for the repository with the name <emphasis>15.6</emphasis> the content of the macro is <code>home:hans / 15.6</code>.
       The repository name is not appended case it is <code>standard</code>.
       If the build happens in the project with the name <emphasis>openSUSE:Factory</emphasis> for the repository with the name <emphasis>standard</emphasis> the content of the macro is <code>openSUSE:Factory</code>.
       This macro is only available for all <xref linkend="cha-obs-package-formats"/> during build time.
```

Reply:
I would also move this `listitem` up, below `%_repository` as it's clearly related.